### PR TITLE
Update aws_sdk to 0.10

### PR DESCRIPTION
**Pull request by sciguy16**
Hello, I've updated aws_sdk to 0.10 and removed 0.7. I've also fixed the failing doctests

### CODE REVIEW

**Comment by bryanburgers** [comment]
Thank you! Looks awesome. 

Two requests:

* Keep 0.7 around. As a 1.0 library, we’ve made a stabilitY guarantee. For users using 0.7, it would be very unexpected to have support yanked from underneath them in a minor point release.

* Don’t bump the version number. That’s typically done in a separate commit that gets tagged, for tracking.

**Comment by sciguy16** [comment]
Thanks for the quick review! I've put 0.7 back in and split the `__doc` array in Cargo.toml onto multiple lines to make it a bit more manageable

File: Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ keywords = ["serde", "rusoto", "dynamodb", "dynamo", "serde_dynamodb"]
 __aws_sdk_dynamodb_0_7 = { package = "aws-sdk-dynamodb", version = "0.7", default-features = false, optional = true }
 __aws_sdk_dynamodb_0_8 = { package = "aws-sdk-dynamodb", version = "0.8", default-features = false, optional = true }
 __aws_sdk_dynamodb_0_9 = { package = "aws-sdk-dynamodb", version = "0.9", default-features = false, optional = true }
+__aws_sdk_dynamodb_0_10 = { package = "aws-sdk-dynamodb", version = "0.10", default-features = false, optional = true }
```
Comment (mbergkvist) [inline Cargo.toml:18]: Perhaps bump `aws-sdk-dynamodbstreams-0.10` too?

File: src/lib.rs [hunk 2]
```diff
@@ -232,12 +232,6 @@ use macros::{aws_sdk_macro, aws_sdk_streams_macro, rusoto_macro, rusoto_streams_
 pub use ser::{to_attribute_value, to_item, Serializer};
 pub use test_attribute_value::TestAttributeValue;
 
-aws_sdk_macro!(
```
Comment (mbergkvist) [inline src/lib.rs:235]: Re-add this too?

Comment (sciguy16) [reply src/lib.rs]: Good spot, thanks!

**Comment by bryanburgers** [comment]
Released as 3.0.3! Thanks!

**Pull request merged**